docs/tasks/access-kubernetes-api/extend-api-third-party-resource.md

```diff
@@ -14,13 +14,24 @@ redirect_from:
 
 ## What is ThirdPartyResource?
 
-**WARNING: ThirdPartyResources are not yet considered stable, and the API and/or storage could change before GA.
-Development and outstanding issues are tracked at [https://github.com/kubernetes/features/issues/95](https://github.com/kubernetes/features/issues/95).**
+**WARNING: ThirdPartyResources will be removed in 1.8 without access to existing data!  See the section on migration to CustomResourceDefinitions.**
```

Comment:
Given a more complete implementation for CRD, continued problems for TPR, and a semi-automatic migration from TPR to CRD, I think we can remove this in 1.8.

@erictune @enisoc @kubernetes/sig-api-machinery-misc